tests/utils/LinearAlg2DTest.cpp

```diff
@@ -62,54 +57,56 @@ TEST_P(GetDist2FromLineSegmentTest, GetDist2FromLineSegment)
     int16_t supposed_is_beyond = 0;
     const coord_t supposed_distance = LinearAlg2D::getDist2FromLineSegment(line_start, point, line_end, &supposed_is_beyond);
     ASSERT_LE(std::fabs(sqrt(double(supposed_distance)) - sqrt(double(actual_distance2))), maximum_error)
-            << "Line [" << line_start.X << ", " << line_start.Y << "] -- [" << line_end.X << ", " << line_end.Y << "], point [" << point.X << ", " << point.Y << "], squared distance was " << supposed_distance << " rather than " << actual_distance2 << ".";
+      << "Line [" << line_start.X << ", " << line_start.Y << "] -- [" << line_end.X << ", " << line_end.Y << "], point [" << point.X << ", "
```

Comment:
Indent with 4 spaces (also below).

This line could definitely be broken up better than what it is now, halfway through the printing of a coordinate.

Reply:
see comment about fmt and mark it as fixme